gnovm/cmd/gno/transpile.go

```
@@ -259,8 +292,116 @@ func goBuildFileOrPkg(fileOrPkg string, cfg *transpileCfg) error {
 	goBinary := cfg.goBinary
 
 	if verbose {
-		fmt.Fprintf(os.Stderr, "%s\n", fileOrPkg)
+		fmt.Fprintf(os.Stderr, "%s [build]\n", filepath.Clean(fileOrPkg))
+	}
+
+	return buildTranspiledPackage(fileOrPkg, goBinary)
+}
+
+// getPathsFromImportSpec returns the directory paths where the code for each
+// importSpec is stored (assuming they start with [transpiler.ImportPrefix]).
+func getPathsFromImportSpec(rootDir string, importSpec []*ast.ImportSpec) (dirs []string) {
+	for _, i := range importSpec {
+		path, err := strconv.Unquote(i.Path.Value)
+		if err != nil {
+			continue
+		}
```

IMHO these swallowed errors eventually cause problems. I would return ([]string, error)`